project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
+
+Current squad boards:
+
+- [Snappy squad tasks](https://app.zenhub.com/workspace/o/ubuntudesign/snappy-squad-tasks/boards)
```

Comment:
This isn't the snappy board. Snappy is in the `CanonicalLtd` org.

Reply:
https://github.com/canonicalltd/snappy-design-squad

Reply:
Nb Prbblem

Reply:
dbne